live/core/README.md

```diff
@@ -102,16 +102,44 @@ vpc_id=$(aws ssm get-parameter --name "$vpc_id_parameter_name" --query 'Paramete
 echo "VPC ID: $vpc_id"
 ```
 
-### Connecting to the Bastion Host
+## Connecting to the Bastion Host
 
-To establish a connection with the bastion host, use the following commands:
+To establish a secure connection with the bastion host, follow these steps:
+
+### 1. Obtain Required Information
```

Comment:
same here. Also we need to choose one: headers or enumaration. Not both. Probably just using `####` should be fine